frontend/src/app/components/wp-card-view/wp-single-card/wp-single-card.component.ts

```diff
@@ -130,4 +130,8 @@ export class WorkPackageSingleCardComponent extends UntilDestroyedMixin implemen
   private attributeHighlighting(type:string, wp:WorkPackageResource) {
     return Highlighting.inlineClass(type, wp.type.id!);
   }
+
+  public wpProjectName(wp:WorkPackageResource) {
```

Comment:
Just a minor remark for code quality: Please move the method below the last public method. Thus we have first all public methods, and at the end of the file all private methods.